src/include/sof/audio/dai_copier.h

```diff
@@ -56,12 +56,9 @@ int dai_zephyr_multi_endpoint_copy(struct dai_data **dd, struct comp_dev *dev,
 int dai_zephyr_unbind(struct dai_data *dd, struct comp_dev *dev, void *data);
 
 struct ipc4_copier_module_cfg;
-int copier_dai_init(struct comp_dev *parent_dev,
-		    const struct comp_driver *drv,
-		    struct comp_ipc_config *config,
-		    const struct ipc4_copier_module_cfg *copier,
-		    struct pipeline *pipeline,
-		    struct ipc_config_dai *dai,
-		    enum ipc4_gateway_type type,
-		    int index, int dai_count);
+struct copier_data;
+int copier_dai_create(struct comp_dev *parent_dev, struct copier_data *cd,
```

Comment:
I would squash this patch with the previous patch, so we dont have to first add copie_dai_init() and then remove it here.

Reply:
ok, it will just make the patch looks very big.